R/nc_make_network1.R

```diff
@@ -8,53 +8,66 @@
 #' @export
 #'
 #' @examples
-nc_make_network1 <- function(.data,.alpha) {
+nc_make_network1 <- function(.data,.alpha,.network_variables) {
   
-  
-  est.pcor.skel.DAG.adj<-function(dat,alpha_val){
+
+  est.pcor.skel.DAG.adj<-function(dat,alpha_val,.network_variables){
     
     #dat: samples x metabolites data matrix
     
     library(ppcor)
     library(pcalg)
     library(igraph)
     pCor_mat<-ppcor::pcor(dat)$estimate                             #estimate Pearson's partial correlation coefficients
-    colnames(pCor_mat)<-colnames(dat)
-    rownames(pCor_mat)<-colnames(dat)
-    
-    n_samples<-nrow(dat)                                            #number of samples
-    V_met<-colnames(dat)                                            #labels equal to node names i.e. metabolites
-    
-    skel_est<-skeleton(suffStat = list(C=cor(dat),n = n_samples),     #estimate order-independent "PC-stable" skeleton of DAG using PC-algorithm
-                       indepTest = gaussCItest,                        #test conditional independence of Gaussians via Fisher's Z
-                       labels = V_met, method = "stable", alpha = alpha_val, fixedGaps = NULL, fixedEdges = NULL, verbose = FALSE)
+    colnames(pCor_mat)<-colnames(data.frame(dplyr::select(dat,noquote(.network_variables))))
```

Comment:
You don't need `noquote`. Latest versions of dplyr can take the string as is. And if we do need it, using `rlang` and tidy eval should be preferred.